CONTRIBUTING.md

```diff
@@ -133,7 +133,12 @@ issues have a blue `category: *` label.
 *   **documentation** - related to updating / fixing / clarifying
     documentation or guides
 
+### Code Review / Comments
 
+If you'd like to review/comment on code in commits or pull requests, please make
+sure that your suggestions don't alter the functionality.  Someone has taken the
```

Comment:
I don't understand what this is saying, how are suggestions altering functionality?

Reply:
If someone asks for code segment X to change to Y, but Y breaks the code...

Reviews often involve style changes.  I can recall times when I came across old code and thought a 'style change' might clean it up, only to find that the change broke it.  Maybe that's never happened to you...

Feel free to close, not a PR that I have strong feelings about.  Just thought it would be a good reminder.